CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 0.9.0
+
+## Features
+
+- CLI has a new `local-build` command.
+- Packit learned how to look for RPM spec files on its own, so specifying `specfile_path` in the configuration is not mandatory anymore.
+- Default configuration has `tests` jobs enabled from now on: it should simplify configuring Packit in new repositories.
```

Comment:
Maybe worth mentioning, that you need to use tmt format for test definition. Otherwise, we test only the installation.

Reply:
Not sure I understand.
I say: You don't have to define it because it's enabled by default
You say: You have to use `fmt` for defining it

Can you just suggest how you would phrase it?

Reply:
It's a bit complicated to describe.

This just enables the job and then they need to use fmf to create some tests. Otherwise, testing-farm only test the success of the installation.

Reply:
We don't have a crystal ball and don't know their test command/setup.

Reply:
got it, changed